docs/DeveloperGuide.md

```diff
@@ -291,10 +296,10 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 The Delete Review mechanism is facilitated by `AddressBook`. This feature allows the user to delete a review.
 
-For the command, the feature extends `command`, and is implemented as such:
+For the command, the feature extends `Command`, and is implemented as such:
 * `rdel REVIEW_INDEX`
 
-#### Implementation Flow of Review Deleting feature
+#### Implementation flow of Review Deleting feature
```

Comment:
I would think deleting/editing should be lowercase since these are not class names.

Reply:
Added in latest commit